docs/getting-started/integrations/quick-start.md

```diff
@@ -1,16 +1,16 @@
 ---
 menu-title: Vanilla JS
-meta-title: Vanilla JS CKEditor 5 Installation - Quick Start with npm | CKEditor 5 documentation
-meta-description: Install, integrate and configure CKEditor 5 using Vanilla JS with npm or ZIP.
+meta-title: CKEditor 5 installation - quick start with npm or ZIP | CKEditor 5 documentation
+meta-description: Install, integrate and configure CKEditor 5 using npm or ZIP.
 category: self-hosted
 order: 20
 ---
 
 {@snippet installation/integrations/framework-integration}
 
-# Installing Vanilla JS CKEditor&nbsp;5 using npm or ZIP
+# Installing CKEditor&nbsp;5 using npm or ZIP
```

Comment:
Why are we removing Vanilla from h1, it may affect SEO.

Reply:
IMO, it read badly. But if we target the keyword, I'll bring it back.

Reply:
Here's the difference:

<img width="901" alt="Screenshot 2024-08-20 at 13 05 00" src="https://github.com/user-attachments/assets/66a5f155-56b1-4c80-b3a4-f202516809f1">

<img width="820" alt="Screenshot 2024-08-20 at 13 16 23" src="https://github.com/user-attachments/assets/2e5ea6c3-02da-46a5-8931-4fd93d16e221">